spec/features/applications/new_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'application new page' do
+  describe 'as a visitor' do
+    describe 'when I visit the pet index page' do
+      it 'then I see a link to "Start an Application"' do
+        visit '/pets'
+
+        expect(page).to have_link('Start an Application')
+      end
+
+      it 'when I click the link "Start an Applicaiton" I am taken to the new application page' do
+        visit '/pets'
+
+        click_link 'Start an Application'
+
+        expect(current_path).to eq('/applications/new')
+      end
+    end
+
+    describe 'when I visit the application new page' do
+      it 'when I am taken to the new application page, I see a form' do
+        visit '/applications/new'
+
+        fill_in 'name', with: 'Billy'
+        fill_in 'address', with: '1234 Example Street'
+        fill_in 'city', with: 'Fake City'
+        fill_in 'state', with: 'Faketucky'
+        fill_in 'zip', with: '123654'
+        fill_in 'description', with: 'I would be a good parent because I would.'
+
+        click_button 'Submit Application' # within 'form' do find('#submit').click end
```

Comment:
I think this would be a good spot for us to refactor the within block, I know that we mentioned that in that note but just want to make sure that we didn't forget about this.